api-parent/access-api/src/main/java/com/sap/cloud/environment/servicebinding/api/ServiceBindingMerger.java

```diff
@@ -21,6 +28,17 @@ public class ServiceBindingMerger implements ServiceBindingAccessor
     @Nonnull
     private final EqualityComparer equalityComparer;
 
+    /**
+     * Initializes a new {@link ServiceBindingMerger} instance. It will use the given {@code equalityComparer} to
+     * combine the individual {@link ServiceBinding}s returned from the given {@code accessors}.
```

Comment:
> Also the class name is somewhat misleading, since the class doesn't merge bindings, it merges collections of bindings.

That's a fair point.
Do you have a suggestion for a better name? `ServiceBindingCollectionMerger` is pretty long and verbose ... I'd still consider it as we do have a lot of very verbose and long class names in this project.

> highlight a bit more that the intention of this class is to solve potential conflicts

👍

Reply:
I've created [an issue](https://github.com/SAP/btp-environment-variable-access/issues/43) for improving the name of the `ServiceBindingMerger`.